HandheldCompanion/Views/QuickPages/QuickHomePage.xaml.cs

```diff
@@ -27,13 +33,47 @@ public QuickHomePage(string Tag) : this()
         MultimediaManager.BrightnessNotification += SystemManager_BrightnessNotification;
         MultimediaManager.Initialized += SystemManager_Initialized;
 
-        ProfileManager.Applied += ProfileManager_Applied;
-        SettingsManager.SettingValueChanged += SettingsManager_SettingValueChanged;
+
+        ProcessManager.ForegroundChanged += ProcessManager_ForegroundChanged;
+
+
+    }
+
+    private void ProcessManager_ForegroundChanged(ProcessEx processEx, ProcessEx backgroundEx)

```

Comment:
Not going to review this since it's basically c/p some stuff from another page to kill process on quick home page. 
Don't recommend this code with MVVM at all though